conf/routes

```diff
@@ -47,3 +47,7 @@ GET        /data/:arn                     controllers.Api.data(arn)
 
 # Map static resources from the /public folder to the /assets URL path
 GET        /assets/*file                  controllers.Assets.at(path="/public", file)
+
+GET        /owners                        controllers.OwnerApi.ownerList
+GET        /owners/for-stack              controllers.OwnerApi.ownerForStack
```

Comment:
The point of this endpoint is that you don't know the `id` but the would make the following request `/owners/for-stack?stack=discussion&stage=PROD&app=modtools`